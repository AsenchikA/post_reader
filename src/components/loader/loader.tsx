import classNames from 'classnames';
import React, { FunctionComponent } from 'react';
import styles from './loader.css';

interface ILoaderProps {
  className?: string;
}

export const Loader: FunctionComponent<ILoaderProps> = ({ className }) => (
  <div className={classNames(styles.loader, className)} />
);
