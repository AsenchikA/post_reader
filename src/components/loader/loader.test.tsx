import React from 'react';
import renderer from 'react-test-renderer';
import { Loader } from './loader';

it('Loader renders correcly', () => {
  const tree = renderer.create(<Loader className="test-class" />).toJSON();
  expect(tree).toMatchSnapshot();
});
