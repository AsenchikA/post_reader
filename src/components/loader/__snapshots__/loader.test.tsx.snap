// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correcly 1`] = `
<div
  className="loader test-class"
/>
`;
